### PR TITLE
File platform: implemented workflow (folders and files are generated). 

### PR DESCRIPTION
**Pull request by ZDu-IDM**
Implemented FilePlatform workflow:
- user can run Suite/Experiment (like SlurmPlatform)
- Suite/Experiment/Simulation folder structure with full files got created/generated
- All functions, like get_item, get_files, etc. should work
- Only left work is run experiment (execute batch files)

**Note**: batch files execution will be implemented in next PR

### CODE REVIEW

**Review by devclinton — APPROVED** [review]

**Comment by shchen-idmod** [comment]
Can we add bootstrap install for idmtools_platform_file package?

File: idmtools_platform_file/setup.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    packages=find_packages(),
+    test_suite='tests',
+
+    entry_points="""
+[idmtools_platform]
+idmtools_platform_file = idmtools_platform_file.plugin_info:FilePlatformSpecification
+""",
+    extras_require=extras,
+    url='https://github.com/InstituteforDiseaseModeling/idmtools',
+    version='1.7.5'
```
Comment (shchen-idmod) [inline idmtools_platform_file/setup.py:52]: Please update to 1.7.5+nightly

**Review by shchen-idmod — CHANGES_REQUESTED** [review]

File: idmtools_platform_file/.bumpversion.cfg [hunk 2]
```diff
@@ -0,0 +1,29 @@
+[bumpversion]
+current_version = 1.7.4
```
Comment (shchen-idmod) [inline idmtools_platform_file/.bumpversion.cfg:2]: 1.7.5+nightly

**Review by shchen-idmod — CHANGES_REQUESTED** [review]

File: idmtools_platform_file/idmtools_platform_file/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+"""idmtools file platform.
+
+Copyright 2021, Bill & Melinda Gates Foundation. All rights reserved.
+"""
+# flake8: noqa F821
+__version__ = "1.7.5"
```
Comment (shchen-idmod) [inline idmtools_platform_file/idmtools_platform_file/__init__.py:6]: 1.7.5+nightly

**Review by shchen-idmod — CHANGES_REQUESTED** [review]

File: idmtools_platform_file/setup.py (hunk 1 above)
Comment (ZDu-IDM) [reply idmtools_platform_file/setup.py]: Version updated, thanks.

File: idmtools_platform_file/.bumpversion.cfg (hunk 2 above)
Comment (ZDu-IDM) [reply idmtools_platform_file/.bumpversion.cfg]: Version updated, thanks.

File: idmtools_platform_file/idmtools_platform_file/__init__.py (hunk 3 above)
Comment (ZDu-IDM) [reply idmtools_platform_file/idmtools_platform_file/__init__.py]: Version updated, thanks.

**Comment by ZDu-IDM** [comment]
> Can we add bootstrap install for idmtools_platform_file package?

Done, thanks.

**Comment by shchen-idmod** [comment]
Can you fix lint?

**Comment by ZDu-IDM** [comment]
> Can you fix lint?

Fixed several lint errors, the rest will be fixed in next PR. Thanks!

**Review by shchen-idmod — APPROVED** [review]
Looks good now

**Pull request merged**